src/zenml/models/model_base_model.py

```diff
@@ -58,3 +61,23 @@ class ModelBaseModel(BaseModel):
     tags: Optional[List[str]] = Field(
         title="Tags associated with the model",
     )
+
+
+class ModelConfigModel(ModelBaseModel):
```

Comment:
I moved it out due to another circular import hell.

Reply:
Generally I'd suggest looking into this again to figure out whether there's a cleaner solution to the circular import, but since it's the only open issue on this PR I'd also be fine with just merging it

Reply:
I tried multiple times but will proceed as is now, cause it is just a structuring thing, not an implementation problem. This small move needs too much of an effort and will touch tons of not related code.